Library/Homebrew/cli/named_args.rb

```diff
@@ -98,8 +98,8 @@ def load_formula_or_cask(name, only: nil, method: nil)
               Formulary.factory(name, *spec, force_bottle: @force_bottle, flags: @flags)
             when :resolve
               resolve_formula(name)
-            when :keg
-              resolve_keg(name)
+            when :default_kegs
```

Comment:
@MikeMcQuaid is this going to be a backward-incompatible change? It looks like `Homebrew::CLI::NamedArgs` is in the private API but [`Homebrew::CLI::Args#named`](https://rubydoc.brew.sh/Homebrew/CLI/Args.html#named-instance_method) is a public method that returns a `NamedArgs` array. I wonder, then, if the `@api private` label here is incorrect.

In fact, I realized that I'm using `args.named.to_formulae` in a few taps, not realizing it wasn't a public method.

Reply:
I think this is. Ideally we'd allow but deprecate (on the next minor version e.g. add a commented out `# odeprecated`) the existing naming.

Reply:
@MikeMcQuaid should the `@api` level be changed?